experiments/spotify/2_consume_source/README.md

```diff
@@ -0,0 +1,20 @@
+# Pipeline that consume source
+The project structure may be created as follows:
+
+1. by cloning github template
+2. with an empty folder - in that case we run on defaults
+3. with `dlt init pipeline [pipeline_name]` see also README for `consume_source` case
+
+In our case we do
+```
+dlt init consume spotify_pipeline
+```
+
+which will create folder structure and `spotify_pipeline.py` with necessary imports and maybe some code commented out...
+
+## Running the pipeline
```

Comment:
what about credentials? in `.dlt/credentials`?

Reply:
yes, or any other name we pick